docker/Dockerfile

```diff
@@ -1,11 +1,13 @@
 # syntax=docker/dockerfile:1
-FROM golang:1.18.6 as builder
```

Comment:
@iogakos I can not see where do we pass the following arguments during build
- BUILDPLATFORM
- TARGETARCH
- BUILDOS

Reply:
These are automatically passed by docker build extensions.

https://docs.docker.com/engine/reference/builder/#automatic-platform-args-in-the-global-scope